rioxarray/rioxarray.py

```diff
@@ -255,17 +320,23 @@ def crs(self):
             Retrieve projection from `xarray.DataArray`
         """
         if self._crs is not None:
-            return self._crs
+            return None if self._crs is False else self._crs
```

Comment:
When is `self._crs` False?

Reply:
It is `False` when it has gone through the process of looking for the CRS in all of the expected locations and failed. This prevents it from re-checking every time when the CRS is not found from attributes in the dataset.

Reply:
Nice.